bootable-jar/WFLY-14411_Bootable_Jar_Upgrade.adoc

```diff
@@ -85,17 +94,23 @@ are in use for the upgrade.
 * This mechanism strongly depends on Maven dependency resolution. 
 Any artifact to upgrade must be present in accessible Maven repositories (remote or local cache).
 
-* An artifact present in the  `overridden-artifacts` set must be present in the dependencies section. 
+* An artifact present in the  `overridden-server-artifacts` or `overridden-artifacts` set must be present in the dependencies section. 
 It is advised that the dependency scope be `provided` to be not included in the application artifact (eg: war).  A warning is displayed if
 the scope is not `provided`.
 If an artifact is not present in the dependencies, then a failure occurs during build.
 
-* An artifact is referenced in the `overridden-artifacts` by GroupId, ArtifactId and optionally Classifier. 
+* An artifact is referenced in the `overridden-server-artifacts` by GroupId, ArtifactId and optionally Classifier. 
 Version being retrieved from the Maven dependencies. 
 
-* An overridden artifact (artifact present in the `overridden-artifacts` list) must be unique. Any duplicate will make the packaging to fail.
+* An artifact is referenced in the `overridden-artifacts` by GroupId, ArtifactId, optionally Classifier and optionally Version. 
+Version is being retrieved from the Maven dependencies if not set. 
+
+* An overridden artifact (artifact present in the `overridden-server-artifacts` list) must be unique. Any duplicate will make the packaging to fail.
+
+* An overridden artifact (artifact present in the `<FeaturePack>` `overridden-artifacts` list) must be unique. Any duplicate will make the packaging to fail.
 
-* Adding an overridden artifact that is not part of the provisioned server artifacts will lead to a failure during build.
+* Adding an overridden artifact that is not part of the provisioned server artifacts will lead to a failure during build. If the overridden artifact is associated
+to a feature-pack, the check is done in the context of the feature-pack.
```

Comment:
Assuming what I say is below is, perhaps add something like this after the first sentence in this paragraph:

"If an overridden artifact configuration includes an optional Classifier setting, the provisioned server artifact must have the same Classifier."

Reading this I realized the existing text before the added scoping commit wasn't clear about whether overriding could replace an artifact with one with a different classifier.

Reply:
Thank-you, added.